### DISCUSSION TITLE
Welcome to MidJourney-Styles-and-Keywords Discussions!

### DISCUSSION THREAD

**Discussion by willwulfken** [willwulfken/MidJourney-Styles-and-Keywords-Reference#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to share ideas, ask questions, and much more.

**Comment by 773433125** [comment]
![inbound1509007440405955900](https://user-images.githubusercontent.com/122175216/211203725-c03b4b47-cf93-417c-a197-997ece90452e.jpg)
خلفيه خياليه اسطورسه شرسه قديمه

**Reply by willwulfken** [reply]
Please translate. I have limited "other language brain cells"

**Reply by 1Fix** [reply]
> Please translate. I have limited "other language brain cells"

Same for me... Google translate does a sort of translation to: 

Fictional backgrounds legendary ferocious old... 

Yep I'm none the wiser

**Comment by 773433125** [comment]
![inbound1044349412178577677](https://user-images.githubusercontent.com/122175216/211203870-9baf5017-0676-4446-bc2b-b3a1011f6b2e.jpg)
Realistic background, terrifying drawing

**Comment by Trick665** [comment]
Greetings..  First off I would like to take the time to compliment you on your prompt page... Absolutely inspirational and informative... You seem to have a pretty firm grasp of midjourney and I was wondering if you had the answer to this question that has been plaguing me for the past few weeks.. How do you get mid Journey to produce an actual word.. I'm trying to design a logo for a nonprofit group we just established... How do I get the actual name of the foundation to be in the logo... It seems to just print random letters.. Guessing how difficult this information is to find on the Internet that it may not be possible.. Thanks a million... Patrick

**Reply by willwulfken** [reply]
It is quite difficult and takes some trial and error. There is also a chance that you won't get it to work.

Using just a prompt only works once in a while and for very short words.

Using an image with the word can work sometimes but with very limited luck.

The best option is using photoshop to add the word into a generated image.
